### DISCUSSION TITLE
Random daily wake ups

### DISCUSSION THREAD

**Discussion by LedgerLand** [teslamate-org/teslamate#1285] [Q&A]
Hey

So, I seem to be having an issues with my Model 3 waking up for no obvious reason. When this does happen it can to over 4+hrs for it to go back to sleep which drains 2-3% of charge, not a big deal but a little frustrating when I'm not going out.
There does not seem to be a pattern to it, so far as I can see. I have checked all of the car settings and there is nothing there that would keep it awake or wake it up.

I have checked the logs from TeslaMate to try and trace out the issue and I can see there are a number of timeout and unavailable errors. In the logs below, the car came back online at 06:44:26 which is shortly after and "error / :vehicle_unavailable message. The Internet connection at home, which the car connects to, is fine overnight as I have checked the logs. The server running TeslaMate is hosted at Vultr and that is showing it has been available so I'm a bit stumped. As of now, it's been trying to fall asleep for 1hr 24 mins.

I'd appreciate it any assistance on this as I'm not sure where to look. Thanks in advance

teslamate_1  | 2021-02-02 17:18:37.500 [info] Refreshing access token ...
teslamate_1  | 2021-02-02 17:18:38.076 [info] Scheduling token refresh in 5 wk 5 d
teslamate_1  | 2021-02-02 18:02:28.219 [info] GET /
teslamate_1  | 2021-02-02 18:02:28.228 [info] Sent 200 in 8ms
teslamate_1  | 2021-02-02 22:41:05.493 car_id=1 [error] Error / :timeout
teslamate_1  | 2021-02-02 22:41:50.644 car_id=1 [error] Error / :timeout
teslamate_1  | 2021-02-03 06:43:57.374 car_id=1 [error] Error / :vehicle_unavailable
teslamate_1  | 2021-02-03 06:44:28.223 car_id=1 [info] Start / :online
teslamate_1  | 2021-02-03 06:44:28.242 car_id=1 [info] Connecting ...
teslamate_1  | 2021-02-03 06:44:29.025 [info] control:hello – 0
teslamate_1  | 2021-02-03 06:45:38.072 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-02-03 06:46:26.061 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 06:46:30.099 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-02-03 06:47:45.367 car_id=1 [info] Suspending logging
teslamate_1  | 2021-02-03 06:47:56.673 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 06:49:27.125 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 06:51:25.906 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 06:52:56.457 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 06:54:26.803 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 06:56:26.178 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 06:57:56.411 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 06:59:26.646 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:01:26.423 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:02:57.299 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:08:38.074 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:08:48.786 [info] Connection was closed (a:1|t:2|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:09:27.301 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:13:38.604 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:13:39.168 [info] control:hello – 0
teslamate_1  | 2021-02-03 07:17:55.202 car_id=1 [error] Error / :vehicle_unavailable
teslamate_1  | 2021-02-03 07:18:16.357 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:18:34.640 car_id=1 [error] Error / :vehicle_unavailable
teslamate_1  | 2021-02-03 07:18:53.719 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:19:04.463 [info] Connection was closed (a:1|t:2|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:19:06.339 car_id=1 [info] Vehicle is still online. Falling asleep for: 34 min
teslamate_1  | 2021-02-03 07:20:34.672 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:23:52.829 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:24:03.538 [info] Connection was closed (a:1|t:2|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:25:34.961 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:28:52.837 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:29:03.539 [info] Connection was closed (a:1|t:2|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:30:35.454 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:33:53.101 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:34:03.811 [info] Connection was closed (a:1|t:2|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:35:35.389 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:38:42.198 [info] GET /
teslamate_1  | 2021-02-03 07:38:42.205 [info] Sent 200 in 6ms
teslamate_1  | 2021-02-03 07:38:53.122 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:39:03.827 [info] Connection was closed (a:1|t:2|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:40:35.472 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:43:53.379 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:44:04.090 [info] Connection was closed (a:1|t:2|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:44:04.644 [info] control:hello – 0
teslamate_1  | 2021-02-03 07:45:35.713 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:48:53.367 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:48:53.938 [info] control:hello – 0
teslamate_1  | 2021-02-03 07:49:07.373 car_id=1 [info] Vehicle is still online. Falling asleep for: 1 h 4 min
teslamate_1  | 2021-02-03 07:49:37.458 [info] GET /
teslamate_1  | 2021-02-03 07:49:37.463 [info] Sent 200 in 4ms
teslamate_1  | 2021-02-03 07:49:38.715 car_id=1 [info] Resuming logging
teslamate_1  | 2021-02-03 07:51:38.510 [info] GET /
teslamate_1  | 2021-02-03 07:51:38.513 [info] Sent 200 in 3ms
teslamate_1  | 2021-02-03 07:51:42.093 car_id=1 [info] Suspending logging [Triggered manually]
teslamate_1  | 2021-02-03 07:51:53.346 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:51:53.919 [info] control:hello – 0
teslamate_1  | 2021-02-03 07:54:17.227 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:54:27.936 [info] Connection was closed (a:1|t:2|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:56:52.805 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:59:16.754 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 07:59:27.458 [info] Connection was closed (a:1|t:2|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:01:53.406 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:04:17.906 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:04:28.602 [info] Connection was closed (a:1|t:2|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:11:01.614 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:11:02.209 [info] control:hello – 0
teslamate_1  | 2021-02-03 08:11:12.356 [info] Connection was closed (a:1|t:2|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:11:51.863 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:16:03.240 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:16:13.973 [info] Connection was closed (a:1|t:2|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:16:14.538 [info] control:hello – 0
teslamate_1  | 2021-02-03 08:16:52.132 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:21:03.543 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:21:14.251 [info] Connection was closed (a:1|t:2|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:21:51.899 car_id=1 [error] Error / :vehicle_unavailable
teslamate_1  | 2021-02-03 08:21:52.161 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:22:02.863 [info] Connection was closed (a:1|t:2|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:22:22.566 car_id=1 [info] Vehicle is still online. Falling asleep for: 32 min
teslamate_1  | 2021-02-03 08:26:13.621 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:26:52.448 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:27:03.187 [info] Connection was closed (a:1|t:2|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:31:15.060 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:31:52.694 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:32:03.394 [info] Connection was closed (a:1|t:2|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:36:14.141 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:36:52.960 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:37:03.658 [info] Connection was closed (a:1|t:2|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:41:14.401 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:41:52.980 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:42:03.723 [info] Connection was closed (a:1|t:2|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:46:14.825 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:46:53.240 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:47:03.949 [info] Connection was closed (a:1|t:2|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:51:15.450 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:51:53.251 [info] Connection was closed (a:1|t:1|d:1). Reconnecting …
teslamate_1  | 2021-02-03 08:52:03.958 [info] Connection was closed (a:1|t:2|d:1). Reconnecting …

**Comment by adriankumpf** [comment]
TeslaMate does not send any commands, and is not able to wake up your car. There have been a few threads on this topic as well, so I moved the issue to "Discussions".

**Reply by LedgerLand** [reply]
Thanks, that's fair enough. What do the following entries in the above log actually mean? What would cause "error /:vehicle_unavailable" and "reconnecting..." to occur?

teslamate_1 | 2021-02-03 08:21:51.899 car_id=1 [error] Error / :vehicle_unavailable

teslamate_1 | 2021-02-03 08:52:03.958 [info] Connection was closed (a:1|t:2|d:1). Reconnecting …

**Reply by adriankumpf** [reply]
`Error / :vehicle_unavailable` is an error returned by the Tesla API. ` Connection was closed. Reconnecting …` is related to the Tesla Streaming API. Both errors occur when your car cannot be reached, basically.
